catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/event/impl/SubscriptionImpl.java

```diff
@@ -22,13 +22,15 @@
 import ddf.catalog.event.Subscription;
 
 public class SubscriptionImpl implements Subscription {
-    private Filter filter;
+    protected Filter filter;
 
-    private DeliveryMethod dm;
+    protected DeliveryMethod dm;
 
-    private Set<String> sourceIds;
+    protected Set<String> sourceIds;
```

Comment:
`getSourceIds()` returns this `Set` directly. Should it return an immutable version or copy of it instead?